reconcile/utils/oc.py

```diff
@@ -735,6 +735,32 @@ def job_logs(self, namespace, name, follow, output):
         # collect logs to file async
         Popen(self.oc_base_cmd + cmd, stdout=output_file)
 
+    def job_logs_latest_pod(self, namespace, name, follow, output):
+        pods = self.get_items("Pod", namespace=namespace, labels={"job-name": name})
+
+        finished_pods = [
+            pod for pod in pods if pod["status"].get("phase") in {"Failed", "Succeeded"}
+        ]
+        if not finished_pods:
+            raise JobNotRunningError(name)
+
+        latest_pod = sorted(
+            finished_pods, key=lambda pod: pod["metadata"]["creationTimestamp"]
+        )[-1]
+        cmd = [
+            "logs",
+            "--all-containers=true",
+            "-n",
+            namespace,
+            f"pod/{latest_pod['metadata']['name']}",
+        ]
+        if follow:
+            cmd.append("-f")
+        # pylint: disable=consider-using-with
+        output_file = open(os.path.join(output, name), "w", encoding="locale")
+        # collect logs to file async
+        Popen(self.oc_base_cmd + cmd, stdout=output_file)
```

Comment:
Do we care that this `oc` command will never terminate? I'm assuming we do not since the pod will terminate and will take down the command with it...

Reply:
afaik it will terminate as long as `follow = False`, no?

Reply:
ah yes I totally forgot to mention this is specifically if following, ie: `follow=True` 

Reply:
i'm not sure how valueable follow=true really is in this context. i will remove it